static/src/stylesheets/module/email/_form.scss

```diff
@@ -7,6 +7,11 @@
     // adjustments, off grid
     width: calc(100% + 8px);
     margin-left: -4px;
+
+    @include mq($from: mobileLandscape, $until: tablet) {
+        margin-right: 50%;
+        width: calc(50% + 8px);
```

Comment:
i can't recall now, that's why i parenthesised that bit – i'll see if i can dig the ref out. i'd be very interested if you found something, will have another look. not a :-1:, just a caution i guess. it could just be myth-making on my part, but it feels instinctively right too, as you say.